### DISCUSSION TITLE
Help with example

### DISCUSSION THREAD

**Discussion by andy-thomason** [extendr/extendr#385] [Examples and documentation]
I'm planning to do a talk about extendr to some bioinformaticians.

I would like to write a rust program that reads some CSV data and plots the result with ggplot using the R! interface.

I know that @clauswilke and @yutannihilation could help here.

From the Rust side, I'd like to be able to use R capabilities and exchange data via RDS, for example
doing stats on logs from a server.

**Comment by clauswilke** [comment]
What specifically are you asking for? Do you want us to provide you with a short R script that reads in data and makes a plot, and then you translate that to Rust?

**Comment by yutannihilation** [comment]
Happy to help you :) What difficulties do you see to implement such a program?

By the way, I don't feel GitHub issues is a very good place to ask questions and get answers. Can we enable [GitHub Discussions](https://docs.github.com/en/discussions/quickstart) on this repo for this purpose? I also feel uneasy a bit that our README refers to the Discord server but it seems few of the main maintainers are very active there, and Discord is not convenient when there are multiple questions are posted.

I don't know well about the Rust community, but it seems using GitHub Discussions is common among Rust repositories, for example:

- https://github.com/gfx-rs/wgpu/discussions
- https://github.com/tokio-rs/tokio/discussions
